pkgs/development/python-modules/pydeck/default.nix

```diff
@@ -1,11 +1,28 @@
-{ lib, buildPythonPackage, fetchPypi, ipykernel, ipywidgets, pythonOlder, pytestCheckHook, pandas }:
+{ lib
+, buildPythonPackage
+, fetchPypi
+, ipykernel
+, ipywidgets
+, pythonOlder
+, pytestCheckHook
+, pandas
+, jinja2
+, numpy
+, traitlets
+}:
 
 buildPythonPackage rec {
   pname = "pydeck";
   version = "0.7.1";
+  format = "setuptools";
 
   disabled = pythonOlder "3.7";
 
+  preBuild = ''
+    # unused
+    rm pyproject.toml
+  '';
```

Comment:
What in the file is causing problems? I didn't see a problematic line on first sight.